data/xenon1t_1ty.csv

```diff
@@ -1,18 +1,18 @@
-"# XENON Collaboration 2018. arXiv-1805.12562, Phys. Rev. Lett. 121, 111302 ",,,,,,,
-## WIMP mass/GeV , upper limit [cm^2], -2 sigmath percentile UL [cm^2], -1sighmath, 0sigmath, 1sigmath, 2sigmath,
-6,2.49E-44,1.93E-44,2.47E-44,4.64E-44,1.27E-43,2.04E-43  
-8,2.05E-45,1.29E-45,1.71E-45,3.65E-45,7.41E-45,1.15E-44  
-9,9.40E-46,4.89E-46,6.70E-46,1.64E-45,3.08E-45,4.78E-45  
-10,5.39E-46,2.55E-46,3.54E-46,8.81E-46,1.61E-45,2.43E-45  
-15,1.17E-46,3.54E-47,5.79E-47,1.42E-46,2.39E-46,3.74E-46  
-20,5.66E-47,1.76E-47,2.76E-47,6.31E-47,1.13E-46,1.63E-46  
-30,4.05E-47,1.07E-47,1.84E-47,3.85E-47,6.80E-47,9.51E-47  
-40,4.48E-47,9.59E-48,1.51E-47,3.39E-47,6.07E-47,9.54E-47  
-50,5.26E-47,1.03E-47,1.61E-47,3.58E-47,6.06E-47,9.37E-47  
-70,6.79E-47,9.82E-48,1.75E-47,4.06E-47,7.09E-47,1.15E-46  
-100,9.12E-47,1.25E-47,2.37E-47,4.97E-47,8.97E-47,1.34E-46  
-150,1.31E-46,1.66E-47,3.34E-47,6.87E-47,1.25E-46,1.83E-46  
-200,1.72E-46,2.48E-47,4.31E-47,9.04E-47,1.50E-46,2.24E-46  
-400,3.34E-46,4.02E-47,8.09E-47,1.69E-46,3.02E-46,4.52E-46  
-1000,8.24E-46,1.00E-46,1.79E-46,4.09E-46,7.11E-46,1.06E-45  
+"#,XENON,Collaboration,2018.,arXiv-1805.12562,Phys.,Rev.,Lett.,121,111302,",,

```

Comment:
Adding commas at the end of a line messes up some .csv parsers-- best not to IMO